### PR TITLE
Player management

### PR DESCRIPTION
**Pull request by Kraysus**
added some player managaement stuff and removed some pointless scripts

### CODE REVIEW

**Review by ShaneWilliamson — APPROVED** [review]
good

File: Assets/Scripts/Player/PlayerManagement.cs [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+	
+	// Update is called once per frame
+	void FixedUpdate () {
+	
+	}
+
+	public int alterAmmo(int change){
+		if (ammo + change > 0) {
+			ammo += change;
+			return ammo;
```
Comment (ShaneWilliamson) [inline Assets/Scripts/Player/PlayerManagement.cs]: Remove these first returns, they're redundant.

**Review by ShaneWilliamson — CHANGES_REQUESTED** [review]
See above

File: Assets/Scripts/Player/PlayerManagement.cs (hunk 1 above)
Comment (Kraysus) [reply Assets/Scripts/Player/PlayerManagement.cs]: yup, did that

**Review by ShaneWilliamson — APPROVED** [review]

**Pull request merged**